src/finite_elasticity_routines.f90

```diff
@@ -4008,6 +4031,14 @@ SUBROUTINE FINITE_ELASTICITY_GAUSS_CAUCHY_TENSOR(EQUATIONS_SET,DEPENDENT_INTERPO
           lambda_a=UP-(UP-LOW)/2.0_DP
 
           F_a_inv=0.0_DP
+          IF(lambda_a<TOL) THEN
+           LOCAL_ERROR="Warning: lambda_a is close to zero"
+           CALL FlagError(LOCAL_ERROR,ERR,ERROR,*999)
+!            WRITE(*,*) "UP: ", UP
```

Comment:
This will not do what I think you want it to do. The FlagError call will automatically trigger an error exeception and the program will stop. I think you want

CALL FlagWarning("Lambda_a is close to zero.",err,error,999)

(which also automatically adds the Warning: part at the beginning).

This will output the warning but not cause the program to stop.